### PR TITLE
Added print message to import uncategorized.sql file manually

### PR DESCRIPTION
**Pull request by sanyamsinghal**

### CODE REVIEW

File: yb-voyager/cmd/importSchema.go [hunk 1]
```diff
@@ -73,6 +74,11 @@ func importSchema() {
 	payload.TargetDBVersion = targetDBVersion
 
 	if !flagPostImportData {
+		filePath := filepath.Join(exportDir, "schema", "uncategorized.sql")
+		if utils.FileOrFolderExists(filePath) {
+			color.Red("\nNote: Voyager doesn't support sql statements mentioned in %q, required to be reviewed and imported manually!!", filePath)
```
Comment (amit-yb) [inline yb-voyager/cmd/importSchema.go:79]: Let's rephrase it as:
```
IMPORTANT NOTE: Please, review and manually import the DDL statements from the uncategorized.sql 
```

File: yb-voyager/src/srcdb/pg_dump_extract_schema.go [hunk 2]
```diff
@@ -159,7 +160,9 @@ func parseSchemaFile(exportDir string) int {
 	if uncategorizedSqls.Len() > 0 {
 		filePath := filepath.Join(schemaDirPath, "uncategorized.sql")
 		// TODO: add it to the analyze-schema report in case of postgresql
-		utils.PrintAndLog("\nSome uncategorized sql statements are present in %q, Needs to review and import them manually!!", filePath)
+		msg := fmt.Sprintf("\nNote: Voyager doesn't support sql statements mentioned in %q, required to be reviewed and imported manually!!", filePath)
```
Comment (amit-yb) [inline yb-voyager/src/srcdb/pg_dump_extract_schema.go:163]: Same as above.

Comment (sanyamsinghal) [reply yb-voyager/src/srcdb/pg_dump_extract_schema.go]: done

File: yb-voyager/cmd/importSchema.go (hunk 1 above)
Comment (sanyamsinghal) [reply yb-voyager/cmd/importSchema.go]: done

**Review by amit-yb — APPROVED** [review]
LGTM.

**Pull request merged**